### PR TITLE
[#10] Add ability to specify multiple fields and change values of created fields

### LINKED ISSUE #10: Add ability to specify multiple fields and change values of created fields
Let's say we have the following mesage:

```protobuf
message SearchRequest {
  required string foo = 1;
  optional int bar = 2;
  repeated int baz = 3;
}
```

TUI should support creating message with this structure and validate against `MessageSchema` (`MessageSchema` for this message will be hardcoded for now) outputting error if it's there. And then, after the message is created, it should print serialized representation of the `SearchRequest`.

Blocked by #9

### PR DESCRIPTION
**Pull request by TejasSC**
Resolves #10

### CODE REVIEW

**Comment by TejasSC** [comment]
screen shot 2018-09-03 at 5 07 58 pm
screen shot 2018-09-03 at 5 07 44 pm
screen shot 2018-09-03 at 5 07 24 pm

**Comment by TejasSC** [comment]
screen shot 2018-09-03 at 5 13 22 pm

**Comment by TejasSC** [comment]
screen shot 2018-09-04 at 3 24 02 pm
screen shot 2018-09-04 at 3 29 06 pm

File: src/Tui.hs [hunk 1]
```diff
@@ -98,12 +102,21 @@ msgSchema = MessageSchema { msgSchemaName = MN "SearchRequest"
...
 
-processFTV :: String -> String -> FieldValue
+data FieldChoice = FReq | FOpt | FRep | Wrong deriving (Show, Read)
+
+toFC :: String -> FieldChoice
+toFC s = case s of
+    "query"    -> FReq
+    "maybeNum" -> FOpt
+    "numList"  -> FRep
+    _          -> Wrong
```
Comment (ChShersh) [inline src/Tui.hs]: I've created issue to change behavior of this function in future:

* https://github.com/Holmusk/proto-route/issues/15

File: src/Tui.hs [hunk 2]
```diff
@@ -98,12 +102,21 @@ msgSchema = MessageSchema { msgSchemaName = MN "SearchRequest"
...
+
+processFTV :: FieldChoice -> String -> Either String FieldValue
 processFTV field value = case field of
-    "query"    -> FText (Req $ read value)
-    "maybeNum" -> FInt  (Opt $ read value)
-    "numList"  -> FInt  (Rep $ read value)
-    _   -> FText Error
+    FReq  -> readEither $ show (FText (Req $ pack value))
+    FOpt  -> readEither $ show (FInt  (Opt $ read value))
```
Comment (ChShersh) [inline src/Tui.hs]: This looks weird. I expect this function to behave something like:

```haskell
FOpt -> FInt . Opt <$> readEither value
```

And similar for other cases

**Pull request merged**